.github/workflows/test_gcp_integration.yaml

```diff
@@ -0,0 +1,92 @@
+name: test-gcp-integration
+
+on:
```

Comment:
Can you add a new on `pull_requests` to test it the entire end-to-end test? We can remove it after we validate it's working

Reply:
I just saw this comment: https://github.com/nebari-dev/nebari/issues/2018#issuecomment-1753454849